jwst/saturation/tests/test_saturation.py

```diff
@@ -195,7 +195,7 @@ def test_nans_in_mask(setup_nrc_cube):
     ngroups = 5
     nrows = 2048
     ncols = 2048
```

Comment:
Also, the test data could be massively scaled down.  20 instead of 2048?  And edit pixel [5,5]?

Reply:
Done